pwa/src/components/translations/translationForm.tsx

```diff
@@ -100,54 +99,60 @@ export const TranslationForm: React.FC<TranslationFormProps> = ({id}) => {
               <div className="row">
                 <div className="col-12">
                   {showSpinner === true ? (
-                    <Spinner/>
+                    <Spinner />
                   ) : (
                     <>
-                      {loadingOverlay && <LoadingOverlay /> }
-                      <div className="row">
-                        <div className="col-6">
-                          <div className="form-group">
-                            <GenericInputComponent
-                              type={"text"}
-                              name={"translationTable"}
-                              id={"translationTableInput"}
-                              data={translation && translation.translationTable && translation.translationTable}
-                              nameOverride={"Table"}
-                              required/>
-                          </div>
-                        </div>
-                        <div className="col-6">
-                          <div className="form-group">
-                            <GenericInputComponent
-                              type={"text"}
-                              name={"language"}
-                              id={"languageInput"}
-                              data={translation && translation.language && translation.language}
-                              nameOverride={"Language"}/>
+                      {loadingOverlay && <LoadingOverlay />}
+                      {
+                        tableName && tableName === 'new' &&
+                        <>
+                          <div className="row">
+                            <div className="col-12">
+                              <div className="form-group">
+                                <GenericInputComponent
+                                  type={"text"}
+                                  name={"translationTable"}
+                                  id={"translationTableInput"}
+                                  data={translation && translation.translationTable && translation.translationTable}
+                                  nameOverride={"Name"}
+                                  required />
+                              </div>
+                            </div>
                           </div>
-                        </div>
-                      </div>
+                          <h4 className="utrecht-heading-4 utrecht-heading-4--distanced">You need to create at least one translation when creating a new table</h4>
```

Comment:
Two notes on this:

1. This heading should be smaller than the Table Title (e.g. _Create Table_) because it's lower in the hierarchy, also, it should be black
2. The ticket mentions a subtitle as well, which is missing.

Reply:
1. This heading is now the exact same size as its parent heading (this is now a `h5` and the table title is a `h4`). However, they both have a font-size of `20`. The `h5` should be smaller (e.g. `font-size: 18px`);
2. You're still missing the subtitle.